docs/dev/release.md

```diff
@@ -0,0 +1,34 @@
+Making releases
+===============
+
+The following examples will assume that we are releasing `v2.0.0`.  Use the correct release version when making the actual release.
+
+Release Hummingbird
+-------------------
+
+    1.  Make a `v2.0.0` tag of hummingbird master:
+        1.  `git checkout master`
+        2.  `git pull upstrem master`
+        3.  `git push origin v2.0.0`
+        4.  `git push --tags`
+    2.  CI will build binaries and create the github release.
+    3.  Edit title and create release notes for the new release at <https://github.com/troubling/hummingbird/releases/tag/v2.0.0>.
+
+Release Hansible
+----------------
+
+    1.  Update `hummingbird_version` to `v2.0.0` in `roles/humminbird-common/defaults/main.yml` and `group_vars/hummingbird/hummingbird` and push to master.
+    2.  Draft a new release at <https://github.com/troubling/hansible/releases/new>.
+    3.  Set Tag version and release title to `v2.0.0`.
+    4.  Fill in release notes and publish the release.
+    5.  Change `hummingbird_version` back to `dev` in `roles/humminbird-common/defaults/main.yml` and push to master.
+
+Release RPC-Hummingbird
+-----------------------
+
+    1.  Update hansible version to `v2.0.0` in `ansible-role-requirements.yml` and push to master.
+    2.  Make a pull request to https://github.com/rcbops/releases:
+        1.  add new version and git sha to `components/rpc-hummingbird.yml`.
+    3.  Once the the change is reviewed and comitted, CI will test and publish the release.
```

Comment:
For now, please have someone in rpc-hummingbird and someone in RE review the change. Once reviewed, add a GitHub comment with `run_component_release` to trigger the release itself.